src/components/Trade/TradeInput.tsx

```diff
@@ -107,22 +145,37 @@ export const TradeInput = ({ history }: RouterProps) => {
           alignItems='center'
           justifyContent='space-between'
         >
-          <IconButton aria-label='Switch' isRound icon={<ArrowDownIcon />} />
+          <IconButton onClick={switchAssets} aria-label='Switch' isRound icon={<ArrowDownIcon />} />
           <Box display='flex' alignItems='center' color='gray.500'>
-            <Text fontSize='sm'>1 BTC = 40,100.45 USDC</Text>
-            <HelperTooltip label='The price is ' />
+            {!quote || action ? (
+              <Text fontSize='sm' translation='trade.searchingRate' />
+            ) : (
+              <>
+                <RawText fontSize='sm'>{`1 ${sellAsset.currency?.symbol} = ${firstNonZeroDecimal(
+                  bn(quote.rate)
+                )} ${buyAsset.currency?.symbol}`}</RawText>
+                <HelperTooltip label='The price is ' />
```

Comment:
Do we need to update this or if we dont have the data yet create an issue to update it?